Troubleshooting guides for disabled allocations (#86789)
elastic/elasticsearch: docs/reference/troubleshooting/data/enable-cluster-allocation.asciidoc

[[allow-all-cluster-allocation]]
== Allow Elasticsearch to allocate the data in the system

The allocation of data in an {es} deployment can be controlled using the <<cluster-routing-allocation-enable, enable 
cluster allocation configuration>>.
In certain circumstances users might want to temporarily disable or restrict the
allocation of data in the system. 

Forgetting to re-allow all data allocations can lead to unassigned shards.

> In order to (re)allow all data to be allocated follow the next steps:

Reviewer: ```suggestion
In order to (re)allow all data to be allocated follow these steps:
```
